Dashboard/app/cljs/src/org/akvo/flow/dashboard/users/users_list.cljs

```diff
@@ -98,9 +98,9 @@
                            {:user user
                             :user-auth-store user-auth-store
                             :roles-store roles-store})}
-     {:title (t> _api_keys)
-      :class "text-center"
-      :component api-user-mark}
+     {:title     (t> _admin)
```

Comment:
There is another string using the `_administrator` translation key, why do we have 2?

Reply:
That key is:

_administrator = Administrator - Create, Modify and Delete Users

which is very long for that column.

Reply:
You're right. Then let's _fix_ the value of the new translation key to: `User Admin` ... not all upper case. The uppercase text transformation is done in the UI via CSS.

Reply:
done